### PR TITLE
GO-3733 Search: do not inject tag/types results for deleted/archived objects

### PR DESCRIPTION
**Pull request by requilence**

### CODE REVIEW

**Review by fat-fellow — CHANGES_REQUESTED** [review]
tests?

**Review by fat-fellow — APPROVED** [review]

**Review by fat-fellow — CHANGES_REQUESTED** [review]
1

**Review by cheggaaa — APPROVED** [review]

**Review by fat-fellow — APPROVED** [review]

File: pkg/lib/localstore/objectstore/queries_test.go [hunk 1]
```diff
@@ -251,14 +263,32 @@ func TestQuery(t *testing.T) {
...
+			bundle.RelationKeyIsDeleted:   pbtypes.Bool(true),
+			bundle.RelationKeyDescription: pbtypes.String("this is a deleted relation's description"),
+			bundle.RelationKeyLayout:      pbtypes.Int64(int64(model.ObjectType_relationOption)),
+		}
+
+		relObjArchived := TestObject{
+			bundle.RelationKeyId:          pbtypes.String("relid3"),
+			bundle.RelationKeyRelationKey: pbtypes.String("bsonid1"),
+			bundle.RelationKeyName:        pbtypes.String("archived"),
+			bundle.RelationKeyIsDeleted:   pbtypes.Bool(true),
```
Comment (jmetrikat) [inline pkg/lib/localstore/objectstore/queries_test.go:279]: @requilence this should be `bundle.RelationKeyIsArchived`?

Comment (requilence) [reply pkg/lib/localstore/objectstore/queries_test.go:279]: nope, it's just a name of the object I use to search :)

**Pull request merged**